.github/workflows/conda-cpp-build.yaml

```
@@ -132,9 +132,12 @@ jobs:
       - name: Telemetry setup
         uses: rapidsai/shared-actions/telemetry-dispatch-setup@main
         continue-on-error: true
-        if: ${{ vars.TELEMETRY_ENABLED == 'true' && github.run_attempt == '1' }}
+        if: ${{ vars.TELEMETRY_ENABLED == 'true' }}
+        env:
+          GH_TOKEN: ${{ inputs.alternative-gh-token-secret-name && secrets[inputs.alternative-gh-token-secret-name] || github.token }}
```

```suggestion
          GH_TOKEN: ${{ github.token }}
```

Just putting this in writing from our offline conversation, so it's here in the history... all of these should just use `${{ github.token }}`. The telemetry stuff only needs to deal with artifacts from the same repo the workflow is running in, so the default repo-scoped token should be sufficient.